fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -763,6 +766,26 @@ private String mintNewPid(final String slug) {
         return pid;
     }
 
+    /**
+     * perform save with checking to prevent SN-Sibling nodes from creating
+     * @param resource the FedoraResource need to save
+     * @throws javax.jcr.AccessDeniedException
+     * @throws VersionException
+     * @throws LockException
+     * @throws ConstraintViolationException
+     * @throws RepositoryException
+     */
+    private synchronized void save(final FedoraResource resource) throws javax.jcr.AccessDeniedException,
```

Comment:
You are already doing this, no?
https://github.com/fcrepo4/fcrepo4/pull/1057/files/ba67f4a3027097bdaae93543c032f3d8a03a2aa0#diff-39785d72ffe23548cc619403f35d9028R2499
and
https://github.com/fcrepo4/fcrepo4/pull/1057/files/ba67f4a3027097bdaae93543c032f3d8a03a2aa0#diff-39785d72ffe23548cc619403f35d9028R2517